### DISCUSSION TITLE
Window is shown like an embosed one when  fltk app is set to frame_type FlatBox

### DISCUSSION THREAD

**Discussion by santokalayil** [fltk-rs/fltk-rs#711] [Q&A] [answered]
Window is shown like an embosed one when  fltk app is set to frame_type FlatBox

```
use fltk::{app, prelude::*, window::Window};
let app = app::App::default();
fltk::app::set_frame_type(fltk::enums::FrameType::FlatBox);
let mut wind = Window::new(100, 100, 400, 300, "Hello from rust");
wind.end();
wind.show();
app.run().unwrap();
```
How to change the look window to flatbox. Extra line `wind.set_frame(fltk::enums::FrameType::FlatBox)` does not make it look like flatbox anymore. 
Or is it some kind of bug?

**Comment by MoAlyousef** [comment, marked as answer]
Can you try building against master:

```toml
[dependencies]
fltk = { version = 1.0.17, git = "https://github.com/fltk-rs/fltk-rs" }
```

I changed the implementation of app::set_frame_type() and have added app::swap_frame_type() with the old implementation.
I'll publish 1.0.17 in a few days.

**Reply by santokalayil** [reply]
Yes. I tried worked. In toml code 1.0.17 has to be '1.0.17'. I fixed it and used it worked well.
